tests/QueueTest.php

```diff
@@ -146,17 +147,22 @@ public function it_posts_the_task_the_correct_queue()
         // Arrange
         CloudTasksApi::fake();
 
+        $closure = fn () => 'closure job';
+        $closureDisplayName = CallQueuedClosure::create($closure)->displayName();
+
         // Act
         $this->dispatch((new SimpleJob()));
         $this->dispatch((new FailingJob())->onQueue('my-special-queue'));
+        $this->dispatch($closure);
+        $this->dispatch($closure, 'my-special-queue');
 
         // Assert
         CloudTasksApi::assertTaskCreated(function (Task $task, string $queueName): bool {
             $decoded = json_decode($task->getHttpRequest()->getBody(), true);
             $command = IncomingTask::fromJson($task->getHttpRequest()->getBody())->command();
 
             return $decoded['displayName'] === SimpleJob::class
-                && ($command['queue'] ?? null) === null
+                && $command['queue'] === 'barbequeue'
```

Comment:
This failed initially after the changes, because `$command['queue']` is now always set.